package.json

```diff
@@ -40,6 +40,7 @@
     "babel-core": "^6.26.3",
     "babel-jest": "^27.2.5",
     "babel-polyfill": "^6.26.0",
+    "css-loader": "^6.4.0",
```

Comment:
@afong888 Are css-loader and style-loader needed if we are using styled-components?